base/deprecated.jl

```diff
@@ -3826,6 +3827,12 @@ end
 
 @deprecate substrides(s, parent, dim, I::Tuple) substrides(parent, strides(parent), I)
 
+
+@deprecate *(A::Factorization, B::Factorization) Matrix(A)*Matrix(B)
+@deprecate *(A::Adjoint{<:Any,<:Factorization}, B::Factorization) adjoint(Matrix(A.parent)) * Matrix(B)
+@deprecate *(A::Factorization, B::Adjoint{<:Any,<:Factorization}) Matrix(A) * adjoint(Matrix(B.parent))
```

Comment:
Are these deprecations related?

Reply:
Good question. I think I added them to deal with the QR/LQ business, but they may no longer be necessary.

Reply:
I think the story was that there were a couple of these methods, but they were inconsistently defined and mostly just did this anyway. It is probably better to define more specific deprecations though.